src/documentation/text/02_Theory/index.tex

```diff
@@ -0,0 +1,19 @@
+Aktoren stellen nebenläufige Einheiten dar, welche nicht über einen gemeinsamen Speicherbereich verfügen, sondern über Nachrichten kommunizieren (Message passing).
+Die Kommunikation basiert auf dem Versenden von Nachrichten zu Empfängern (FIFO-Prinzip).
+Jeder Aktor verfügt dabei über einen Posteingang, eine Adresse und ein Verhalten.
```

Comment:
`Verhalten.` ist hier ein schwieriges Wort. Geht ja auch eher in Richtung, dass ein Aktor je nach Zustand anders reagiert...